### DISCUSSION TITLE
Scooters identified as person from front view

### DISCUSSION THREAD

**Discussion by mannish-kumar** [blakeblackshear/frigate#3902] [Ask A Question]
Hi,

I having difficulty in identifying scooters correctly. Frigate identifies them as person from the front view.

I have another camera installed in different direction and it detects scooters correctly as motorcycle. Although the scooters are not exactly the same but are similar.

**Front View:**
![image](https://user-images.githubusercontent.com/87553561/191975693-ad010d58-ed4e-404f-a84c-5e0b8183b8c3.png)

**Side View**
![snapshot-1663939741 77466-axws7g](https://user-images.githubusercontent.com/87553561/191976401-f821062b-5670-49b9-b9d2-bedd47324704.jpg)

**Comment by NickM-27** [comment]
If those are commonly in the same spot you could use an object mask to mask out person on there. Unfortunately they are a similar size to an actual person.

**Reply by mannish-kumar** [reply]
The position of scooters keeps changing on daily basis and I don't have a control over it. I could mask a larger area and get rid of this issue but again the parking is open from all three sides and any person could jump the wall and come in.

**Reply by NickM-27** [reply]
Most of these scores are pretty low so you may have luck increasing the person score

**Comment by mannish-kumar** [comment]
Yes, it is low because the person near the gate was once identified at 59 score. I didn't want to miss any event so kept it low.

**Comment by mannish-kumar** [comment]
I changed the threshold to 70 but this still continues to bother me. Is there a way I can fix it?

![image](https://user-images.githubusercontent.com/87553561/198049166-69079757-bd85-4e0b-a262-fa09e41f627d.png)

**Reply by Mincka** [reply]
Maybe you can use a zone around the door because I don't see people leaving scooters right in front of the doors.
Then, you would require this zone to trigger a person event.
If the goal is to be notified when people are coming or leaving, it could be enough.

**Reply by mannish-kumar** [reply]
Ok. In that case I guess two zones needs to be created. One near the door and the other one near the wall on right side to track if someone jumps the wall from outside and enters in. 
The person will be detected near these zones (door and wall) but what if he enters into the non zoned area through these zones and stays for a 5 min or 10 min in the non zone area, Will he be still tracked in non zone area?

**Reply by blakeblackshear** [reply]
> Will he be still tracked in non zone area?

Yes
